protocol/x/blocktime/keeper/keeper.go

```diff
@@ -103,6 +103,15 @@ func (k Keeper) UpdateAllDowntimeInfo(ctx sdk.Context) {
 		metrics.BlockTimeMs,
 	)
 
+	metrics.AddSampleWithLabels(
+		metrics.BlockTimeDistribution,
+		float32(delta.Milliseconds()),
+		metrics.GetLabelForStringValue(
+			metrics.Proposer,
```

Comment:
We don't really expect proposer with bad uptime to have consistently high blocktime due to how blocktime is calculated. Nevertheless, emitting label for potential insight